### DISCUSSION TITLE
Add-license-1 branch automatically added to my repository after adding the MIT license to my repository what its means?

### DISCUSSION THREAD

**Discussion by muhibarshad** [community/community#23057] [New to GitHub]
<p>What i should do right know kindly guide me</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Most likely that’s a branch for a PR that adds the license file. If that’s the case, review the PR and (if it’s as you want) merge it.</p>
<p>If you need more help a link to the repository would be important, so people can actually see what’s going on.</p>

**Reply by mochomo2** [reply]
Trial
RXhwKN4sF2sTSD4S

**Reply by mochomo2** [reply]
Que necesito hacer

**Comment by muhibarshad** [comment]
<p>Ok :blush:Thanks .I deleted that branch and all the things well.</p>
